docs/src/main/sphinx/connector/faker.md

```diff
@@ -0,0 +1,290 @@
+# Faker connector
+
+The Faker connector generates random data matching a defined structure. It uses
+the [Datafaker](https://www.datafaker.net/) library to make the generated data
+more realistic. Use the connector to populate another data source with large
+and realistic test data. This allows testing performance of applications
+processing data, including Trino itself, and application user interfaces.
+
+## Configuration
+
+Create a catalog properties file that specifies the Faker connector by setting
+the `connector.name` to `faker`.
+
+For example, to generate data in the `generator` catalog, create the file
+`etc/catalog/generator.properties`. Replace the connection properties as
+appropriate for your setup.
+
+
+```text
+connector.name=faker
+faker.null-probability=0.1
+faker.default-limit=1000
+```
+
+Create tables in the `default` schema, or create different schemas first.
+Reading from tables in this catalog will return random data. See
```

Comment:
```suggestion
Reading from tables in this catalog returns random data. See
```

Reply:
?

Reply:
Whoops, missed that one. I ran `vale` again and it spotted it.